### ISSUE TITLE
Nippy error on cached questions with integers

### ISSUE THREAD

**Issue by paoliniluis** [ClickHouse/metabase-clickhouse-driver#147]
### Describe the bug
When you have a table that has UInt32 fields and question caching is enabled, you get a 
```
{"nippy/unthawable":{"type":"serializable","cause":"quarantined","class-name":"com.clickhouse.data.value.UnsignedInteger","content":"0xACED0005"}}
```
... on those fields

### Steps to reproduce
1. Create a table with a UInt32 field
2. Enable question caching with very low values
3. Make the question to be cached (e.g. put the question multiple times in a dashboard so you overload the DB and it takes several seconds to respond)
4. see the error

### Expected behaviour
See the actual value of the field

### Error log
Above

#### Environment
* metabase-clickhouse-driver version: 1.1.1
* metabase-clickhouse-driver configuration: metabase 1.46-RC2 with Clickhouse, all on docker (https://github.com/paoliniluis/metabase-clickhouse)

Note: talked with our devs and this is what they said:
```
https://github.com/metabase/metabase/blob/master/src/metabase/query_processor/middleware/cache/impl.clj#L40 is an example of how we teach nippy how to freeze/thaw (store a class, restore a cache) from disk.
They return numbers as a com.clickhouse.data.value.UnsignedInteger type. Presumably we have to get a string version from it and then create one from a string
They have those functions:
[toString](https://javadoc.io/static/com.clickhouse/clickhouse-data/0.4.1/com/clickhouse/data/value/UnsignedInteger.html#toString())
[of (essentially fromString)](https://javadoc.io/static/com.clickhouse/clickhouse-data/0.4.1/com/clickhouse/data/value/UnsignedInteger.html#valueOf(java.lang.String))
```

**Comment by dmeremyanin** [comment]
@slvrtrn sorry for bothering but is there any chance this will be fixed soon? Thanks

**Comment by slvrtrn** [comment]
It should be fixed in [1.1.5](https://github.com/ClickHouse/metabase-clickhouse-driver/releases/tag/1.1.5). Please create a new issue if something is still broken.

**Comment by dmeremyanin** [comment]
@slvrtrn just checked and it works like a charm. Thanks a lot for the quick fix!
